code/server/dbscripts/testmatches.sql

```diff
@@ -0,0 +1,5 @@
+insert into user_matches (liker_id, likee_id, likes, createdAt, updatedAt) values (202, 11, 1,  '2016-02-01 03:22:27.833 +00:00', '2016-02-01 03:22:27.833 +00:00');
```

Comment:
Well the point of the test user script was to add random users, not our own, but that's besides the point. If we keep the testusers.sql script the same, this script (testmatches.sql) will only add matches to Jorell's account. For testing purposes, we should all have the ability to easily add matches to all personal account which is why I'm suggesting a variable approach rather than a hard-coded one.